tasks/discover-pdrs/index.js

```diff
@@ -35,6 +37,7 @@ const discoverPdrs = async ({ config }) => {
   const discoveredFiles = await listFiles(
     config.provider,
     config.useList,
+    config.httpRequestTimeout,
```

Comment:
Given the growing number of parameters to this function, it would be easier if they were converted to named parameters.